### PR TITLE
Add ROS Noetic support to `beluga_example`

### PR DESCRIPTION
**Pull request by hidmic**
### Proposed changes

Connected to https://github.com/Ekumen-OS/beluga/issues/95. Builds on top of https://github.com/Ekumen-OS/beluga/pull/218. This patch enables `beluga_example` to build against both ROS 2 and ROS 1 distributions.

#### Type of change

- [ ] 🐛 Bugfix (change which fixes an issue)
- [x] 🚀 Feature (change which adds functionality)
- [x] 📚 Documentation (change which fixes or extends documentation)

### Checklist

- [x] Lint and unit tests (if any) pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] All commits have been signed for [DCO](https://developercertificate.org/)

### CODE REVIEW

File: beluga_example/bags/perfect_odometry.old.bag
Comment (nahueespinosa) [inline beluga_example/bags/perfect_odometry.old.bag:1]: This file is not used in any of the launch files and the name suggests is not meant to be pushed.
Same for:
- beluga_example/bags/perfect_odometry.old2.bag
- beluga_example/bags/perfect_odometry.orig.bag

**Review by nahueespinosa — COMMENTED** [review]
Looks great, just a minor comment!

File: beluga_example/bags/perfect_odometry.old.bag
Comment (hidmic) [reply beluga_example/bags/perfect_odometry.old.bag:1]: Argh, I was clearly not paying attention. That's garbage. Will remove.

Comment (hidmic) [reply beluga_example/bags/perfect_odometry.old.bag:1]: Removed, and rebased.

**Review by nahueespinosa — APPROVED** [review]

**Comment by hidmic** [comment]
Hmm, it failed the upload to `codecov`. Re-running humble CI job. Has this happened before?

**Comment by nahueespinosa** [comment]
> Has this happened before?

First time I see it.

**Pull request merged**